gateway/src/integration-test/resources/http/grafana/query/extract-algo/testMissingAdd/testNullDate/testQuery/expectedResponse.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "target": "openSpaceSensors.Temperature222",
+    "datapoints": [
+      [1477917224866, 8.0],
+      [1477895624866, 43.0]
+    ]
+  }
+]
```

Comment:
Same here, a point of openSpaceSensors.Temperature111 should be returned as well !! You should write test depending on what you expect and never on what your program return !!! I should not have to recheck your tests...